lib/ansiblelint/formatters/__init__.py

```diff
@@ -123,10 +123,11 @@ def format(self, match: "MatchError", colored: bool = False) -> str:
         file_path = self._format_path(match.filename or "")
         line_num = match.linenumber
         rule_id = match.rule.id
+        severity = match.rule.severity
         violation_details = match.message
         return (
             f"::{level} file={file_path},line={line_num}"
-            f"::[E{rule_id}] {violation_details}"
+            f"::[E{rule_id}] [{severity}] {violation_details}"
```

Comment:
Based on how the format looks, what you want is to add `,severity={}` on the first line and avoid polluting the message itself.